src/assets/AssetsController.ts

```diff
@@ -461,14 +508,20 @@ export class AssetsController extends BaseController<
       ) {
         return collectibleContracts;
       }
-      const newEntry: CollectibleContract = {
-        address,
-        description,
-        logo: image_url,
-        name,
-        symbol,
-        totalSupply: total_supply,
-      };
+      /* istanbul ignore next */
+      const newEntry: CollectibleContract = Object.assign({},
```

Comment:
@Gudahtt what do you think about this?
we can have many of these parameters as `null` coming from the API, I didn't want to store those, i can imagina having the majority as null from what I've been getting from the API

Reply:
Ah I see, so the API is returning `null` values for these rather than `undefined`, and you're normalizing them here to be `undefined`? That makes sense.

Reply:
yep, cool cool thanks